### PR TITLE
IOSXR/Added show mfib platform evpn bucket location

### PR DESCRIPTION
**Pull request by brennobrenno**
Added show mfib platform evpn bucket location to IOSXR.

Screen Shot 2020-10-27 at 4 14 05 PM

### CODE REVIEW

File: src/genie/libs/parser/iosxr/tests/test_show_mfib.py [hunk 1]
```diff
@@ -96,5 +96,117 @@ def test_show_mfib_route_summary_full_custom_vrf(self):
...
+        Bundle-Ether1        0x4000660         10       DF     F 
+        Bundle-Ether1        0x4000660         11      NDF     F 
+        ------------------------------------------------------"""}
+
+    golden_parsed_output_1 = {
+        'bucket_id':
+            {0:
+                {'esi_interface': 'Bundle-Ether1',
+                 'handle': '0x4000660',
+                 'stale': False,
```
Comment (tahigash) [inline src/genie/libs/parser/iosxr/tests/test_show_mfib.py:132]: let's take `stale`'s value as is.
```suggestion
                 'stale': 'F',
```

**Review by tahigash — CHANGES_REQUESTED** [review]
reviewed schema and left 1 comment

**Review by tahigash — APPROVED** [review]
schema looks good now.

File: changelog/undistributed/changelog_show_mfib_platform_evpn_bucket_location_20201027195422.rst [hunk 2]
```diff
@@ -0,0 +1,3 @@
+* IOSXR
+* Modified show_mfib.py
```
Comment (choudnik) [inline changelog/undistributed/changelog_show_mfib_platform_evpn_bucket_location_20201027195422.rst:2]: Please fix the indentation

File: src/genie/libs/parser/iosxr/show_mfib.py [hunk 3]
```diff
@@ -123,4 +124,105 @@ def cli(self, vrf='', location='', output=None):
...
+                  'handle': str,
+                  'stale': str,
+                  'state': str
+                  },
+             },
+    }
+
+
+# ==========================================================================
+# Parser for 'show mfib platform evpn bucket location'
```
Comment (choudnik) [inline src/genie/libs/parser/iosxr/show_mfib.py:149]: show mfib platform evpn bucket location {location}

File: src/genie/libs/parser/iosxr/show_mfib.py [hunk 4]
```diff
@@ -123,4 +124,105 @@ def cli(self, vrf='', location='', output=None):
                 vrf_dict['no_sg_routes'] = int(group['no_sg_routes'])
                 continue
 
-        return parsed_dict
\ No newline at end of file
+        return parsed_dict
+
+
+# ==========================================================================
+# Schema for 'show mfib platform evpn bucket location'
```
Comment (choudnik) [inline src/genie/libs/parser/iosxr/show_mfib.py:131]: show mfib platform evpn bucket location {location}

File: src/genie/libs/parser/iosxr/tests/test_show_mfib.py [hunk 5]
```diff
@@ -96,5 +96,118 @@ def test_show_mfib_route_summary_full_custom_vrf(self):
         self.assertEqual(parsed_output, self.golden_parsed_output_3)
 
 
+# ==========================================================================
+# Unittest for 'show mfib platform evpn bucket location'
```
Comment (choudnik) [inline src/genie/libs/parser/iosxr/tests/test_show_mfib.py:100]: show mfib platform evpn bucket location {location}

**Review by choudnik — CHANGES_REQUESTED** [review]

**Comment by brennobrenno** [comment]
Added minor changes!

**Review by ThomasJRyan — APPROVED** [review]

**Comment by ThomasJRyan** [comment]
Looks good. Please fix conflicts and we can merge it in

**Comment by tahigash** [comment]
@brennobrenno I tried to resolve the conflict for you but I couldn't. There might be glitch on GitHub or I'm missing something. If you cannot resolve the conflict even from your side, please create new PR with same files. we will approve and merge on new PR. sorry for inconvenience.

**Comment by brennobrenno** [comment]
> @brennobrenno I tried to resolve the conflict for you but I couldn't. There might be glitch on GitHub or I'm missing something. If you cannot resolve the conflict even from your side, please create new PR with same files. we will approve and merge on new PR. sorry for inconvenience.

Sounds good @tahigash , I will try to resolve conflicts soon. Thanks :)!

**Comment by tahigash** [comment]
> > @brennobrenno I tried to resolve the conflict for you but I couldn't. There might be glitch on GitHub or I'm missing something. If you cannot resolve the conflict even from your side, please create new PR with same files. we will approve and merge on new PR. sorry for inconvenience.
> 
> Sounds good @tahigash , I will try to resolve conflicts soon. Thanks :)!

sorry for many times. my last try resolved conflict and Checks also passed. 

Thanks for contribution and I'm merging this parser.

**Comment by brennobrenno** [comment]
@tahigash Perfect, thank you!

**Pull request merged**